windows/ReactNativeCameraCPP/ReactCameraConstants.h

```diff
@@ -0,0 +1,140 @@
+#pragma once
+
+#include <functional>
+
+namespace winrt::ReactNativeCameraCPP
+{
+    static const int CameraAspectFill = 0;
+    static const int CameraAspectFit = 1;
+    static const int CameraAspectStretch = 2;

```

Comment:
do these need to actually be ints? can't these be values in an enum class? same with others?

Reply:
We could, but I will have to cast it twice. Once from module to RN and once from RN back to module since RN only deal with number. That's why I created a reactconstants.h and defined everything as numeric there.

Reply:
My bad - I read these as being members of a class but they're in a namespace.
I think what you had before was fine. In fact you could just say constexpr int CameraAspectFill = 0; if you wanted to be more explicit that this shouldn't take up any storage.

---
In reply to: [387861878](https://github.com/react-native-community/react-native-camera/pull/2723#discussion_r387861878) [](ancestors = 387861878)